docs/Alethio/Lite-Network-Monitor.md

```diff
@@ -3,21 +3,21 @@ description: Alethio EthStats Lite Network Monitor
 
 # Alethio EthStats Lite Network Monitor
 
-Use the [Alethio EthStats Lite Network Monitor](https://github.com/Alethio?utf8=%E2%9C%93&q=ethstats&type=&language=javascript)
+Use [EthStats Lite](https://github.com/Alethio?utf8=%E2%9C%93&q=ethstats&type=&language=javascript)
```

Comment:
There's still a few instances of 'Network Monitor' on this page that need to be updated to EthStats Lite

Reply:
I wanted to keep the term only once in the title to make sure people know what it's about then call it by the EthStats lite name. Should I add parentheses ?

Reply:
Updated other instances of the name. Just kept it with parentheses in the title and in the description for search engines